pkgs/os-specific/linux/nvidia-x11/generic.nix

```diff
@@ -126,7 +132,9 @@ let
       homepage = "https://www.nvidia.com/object/unix.html";
       description = "X.org driver and kernel module for NVIDIA graphics cards";
       license = licenses.unfreeRedistributable;
-      platforms = [ "x86_64-linux" ] ++ optionals (!i686bundled) [ "i686-linux" ];
+      platforms = [ "x86_64-linux" ]
+        ++ optionals (!i686bundled) [ "i686-linux" ]
+        ++ optionals (sha256_aarch64 != null) [ "aarch64-linux" ];
```

Comment:
Then `!i686bundled` should actually be `sha256_32bit != null`, since `i686bundled` is an overloaded variable that also is false if `disable32Bit` is true.

That would make it accurately reflect what kind of binary packages are available.

Reply:
Would you like me to change the i686bundled to sha256_32bit in this pull request?

Reply:
Yeah I see the problem now, the package which has i686 bundled libraries, does not mean that the whole package would work on 32-bit x86